### PR TITLE
Fixes #10448 - listing stacks

### PR DESCRIPTION
**Pull request by tstrachota**

### CODE REVIEW

File: app/controllers/foreman_deployments/api/v2/stacks_controller.rb [hunk 1]
```diff
@@ -17,6 +17,11 @@ def create
           ForemanDeployments::StackParser.parse(@stack.definition)
           process_response @stack.save
         end
+
+        api :GET, '/stacks/', N_('List saved stacks')
+        def index
+          @stacks = Stack.all
```
Comment (ShimShtein) [inline app/controllers/foreman_deployments/api/v2/stacks_controller.rb]: Do we need to worry about permissions at this stage?

Comment (ShimShtein) [inline app/controllers/foreman_deployments/api/v2/stacks_controller.rb]: If wwe are using the framework, it should reference the `resource_scope` method:

``` ruby
@stacks = resource_scope
```

Comment (tstrachota) [inline app/controllers/foreman_deployments/api/v2/stacks_controller.rb]: Correct, I plan to add permissions later but the resource scope can be already in the place. I'll add it.

**Comment by ShimShtein** [comment]
Merged as https://github.com/theforeman/foreman_deployments/commit/d98341ed94453ffbb41b33b541693541d5822db6

Thanks @tstrachota

**Pull request closed without merging**